library/Vanilla/Formatting/Quill/BlotGroup.php

```diff
@@ -395,11 +396,13 @@ public function getTestData(): array {
      */
     public function getUnsafeText(): string {
         $text = "";
+        $mainBlot = $this->getMainBlot();
+        if ($mainBlot instanceof SpoilerLineTerminatorBlot) {
+            return t("(Spoiler)\n");
```

Comment:
We had no `t()` global function usage in this class before.
Now we are starting to pollute with non-declared dependencies? 
```suggestion
            return "(".t('Spoiler').")\n");
```
Another question. If we detect `SpoilerLineTerminatorBlot` instance. 
Probably that object could/should return some constant value or provide some method to get this text?

Reply:
The actual existing translation already has the parentheses.

https://github.com/vanilla/locales/blob/master/tx-source/site_core.php#L115

I update the method call though.